eo-maven-plugin/src/test/resources/org/eolang/maven/snippets/ifthenelse.yaml

```diff
@@ -1,11 +1,12 @@
-skip: true
 out:
   - ".*greater.*"
-args: [ "main", "17" ]
+file: org/eolang/snippets/ifthenelse.eo
+args: [ "org.eolang.snippets.ifthenelse", "17" ]
```

Comment:
@yegor256 maybe it's not really part of you task, but what's the point passing "17" here, if it isn't used?

Reply:
@maxonfjvipon fixed in 6c24baec6